unit_tests/test_cluster.py

```diff
@@ -408,3 +409,31 @@ def test_monitoring_version(self):
             ("/usr/bin/scylla --build-id", (0, "xxx", "")),
         ))
         self.assertEqual(self.monitor_cluster.monitoring_version, "master")
+
+
+class TestInstallCassandraTools(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.temp_dir = tempfile.mkdtemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls.temp_dir)
+
+    def setUp(self):
+        self.node = DummyNode(name='test_node',
+                              parent_cluster=None,
+                              base_logdir=self.temp_dir,
+                              ssh_login_info=dict(key_file='~/.ssh/scylla-test'))
+        self.db_cluster = DummyDbCluster([self.node])
+        self.monitor_cluster = BaseMonitorSet({"db_cluster": self.db_cluster}, {})
+        self.monitor_cluster.log = logging
+        self.node.remoter = DummyLocalRemote()
```

Comment:
I'm not sure I love the idea that a unittest has the side effect of install something on my computer.
or using sudo

Reply:
OK.
I removing it.